applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -42,6 +42,8 @@ include::modules/sbo-advanced-binding-options.adoc[leveloffset=+1]
 include::modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc[leveloffset=+1]
 include::modules/sbo-unbinding-workloads-from-a-backing-service.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
 == Additional resources
 * xref:../../applications/connecting_applications_to_services/understanding-service-binding-operator.adoc#binding-a-workload-together-with-a-backing-service[Binding a workload together with a backing service].
-* xref:../../applications/connecting_applications_to_services/getting-started-with-service-binding.adoc#connecting-the-spring-petclinic-sample-application-to-the-postgresql-database-service[Connecting the Spring PetClinic sample application to the PostgreSQL database service].
\ No newline at end of file
+* xref:../../applications/connecting_applications_to_services/getting-started-with-service-binding.adoc#connecting-the-spring-petclinic-sample-application-to-the-postgresql-database-service[Connecting the Spring PetClinic sample application to the PostgreSQL database service].
+* xref:../..//operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-creating-custom-resources-from-file_crd-managing-resources-from-crds[Creating custom resources from a file]
```

Comment:
```suggestion
* xref:../../operators/understanding/crds/crd-managing-resources-from-crds.adoc#crd-creating-custom-resources-from-file_crd-managing-resources-from-crds[Creating custom resources from a file]
```
Noticed this as I peeked at the build failure.

Reply:
Thanks @maxwelldb .. this is fixed now.